acme/acme/standalone.py

```diff
@@ -83,9 +83,15 @@ def __init__(self, ServerClass, server_address, *remaining_args, **kwargs):
                 new_address = (server_address[0],) + (port,) + server_address[2:]
                 new_args = (new_address,) + remaining_args
                 server = ServerClass(*new_args, **kwargs) # pylint: disable=star-args
-            except socket.error:
-                logger.debug("Failed to bind to %s:%s using %s", new_address[0],
+                logger.debug("Successfully bound to %s:%s using %s", new_address[0],
                     new_address[1], "IPv6" if ip_version else "IPv4")
+            except socket.error:
+                if self.servers:
+                    # Already bound using IPv6.
+                    logger.debug("Failed to bind to %s:%s using %s, but continuing as we've bound using %s",           new_address[0], new_address[1], "IPv6" if ip_version else "IPv4", "IPv6")
```

Comment:
Linter fails here because of the missing newline and indentation. In addition to that, I think we could be more informative here with error message like:

```
Certbot wasn't able to bind to %s:%s using %s, this is often expected due to the dual stack nature of IPv6 socket implementations.
```
I speak only from the personal experience, but seeing `Failed` anywhere in the log entries often takes full focus.

Reply:
The suggestion sounds good! It makes sense to not have `failed` if it'll continue okay.
I'll make the changes tomorrow and ping you.